### ISSUE TITLE
Grid doesn't fill available height

### ISSUE THREAD

**Issue by thomaslindstrom** [apexcharts/apexcharts.js#1569]
# Bug report

## Codepen

https://codepen.io/thomaslindstr_m/pen/JjYygRo?editors=0010

## Explanation

When removing all configured (supposed) paddings and margins, the grid doesn't fill available height. What's weird though, is that with the same configuration, but a different data set, it sometimes does fill the available height. I did not, however, succeed in reproducing that in the codepen, but I've attached screenshots from the production implementation.

<img width="800" alt="Screenshot 2020-04-30 at 17 28 08" src="https://user-images.githubusercontent.com/2833514/80729023-f6cec780-8b07-11ea-9bd7-8af78a7bb644.png">
<img width="796" alt="Screenshot 2020-04-30 at 17 28 17" src="https://user-images.githubusercontent.com/2833514/80729050-fc2c1200-8b07-11ea-8b3c-ae4aedae2627.png">

tl;dr: I want the chart to always fill the available height.

**Comment by thomaslindstrom** [comment]
The space is consistently 5px.

**Comment by thomaslindstrom** [comment]
This was due to `rotateAlways` on `xaxis` being set to `false`. Setting it to `true` ensures full height.

I had set `xaxis.labels.show` to `false`, to this behavior was unexpected.

**Comment by junedchhipa** [comment]
Thanks for providing extra details that setting `rotateAlways` produce this issue.
I will take a closer look soon.

**Comment by junedchhipa** [comment]
Use this
```js
{
  chart: {
    sparkline: {
      enabled: true
    } 
  }
}
```
